config/secrets.yml.dev

```diff
@@ -78,12 +78,12 @@ dev_settings: &DEV_SETTINGS
   custom_indices:
     elasticsearch:
       reader:
-        host: http://localhost:9200
+        host: http://localhost:9256
         user: elastic
```

Comment:
going forward are we using 9256 port?

Reply:
Yes - that is the port for the 5.6 cluster in CircleCi.